src/components/Category/Category.style.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+export const CategoryWrapper = styled.section`
+  justify-content: center;
+  display: flex;
+
+  width: 100vw;
+  height: 23.6rem;
+  padding-top: 1rem;
+
+  background-color: ${({ theme: { colors } }) => colors.grayScale.gray2};
+`;
+
+export const ListItemContainer = styled.div`
+  width: 13.6rem;
+  height: 14.8rem;
+  margin: 0 0.25rem;
+
+  cursor: pointer;
+`;
+
+export const ListName = styled.p`
+  width: 12rem;
+  text-align: center;
+  margin: 0rem 0.8rem;
+
+  color: ${({ theme: { colors } }) => colors.grayScale.gray8};
+
+  font: ${({ theme: { fonts } }) => fonts.body3};
```

Comment:
옹 font-family 말고 font라고 써도 잘 적용되나용!?
아니면 아예 요렇게도 가능 !
```suggestion
  ${({ theme: { fonts } }) => fonts.body3};
```